packages/interactivity/src/directives.tsx

```diff
@@ -632,5 +632,5 @@ export default () => {
 		{ priority: 20 }
 	);
 
-	directive( 'each-child', () => null );
+	directive( 'each-child', () => null, { priority: 4 } );
```

Comment:
I think it's a good idea to increase the priority to avoid the creation of inner Directive components. 

But why use 4? Shouldn't it be the minimum possible, 1?

Reply:
5 is the minimum used right now. I added 4 in case there is a future directive that would need more priority.

Reply:
I don't think we need to give more priority to another directive that is going to process something that is just going to be deleted 😄

I was thinking more about when custom directives exist. 4 works well for now, but I think it's better to use 1 anyway.